Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -393,6 +394,7 @@ extension YearInReviewCoordinator: YearInReviewCoordinatorDelegate {
                 visibleVC.present(activityController, animated: true, completion: nil)
             }
         case .dismiss(let isLastSlide):
+            (self.navigationController as? RootNavigationController)?.turnOffForcePortrait()
             navigationController.dismiss(animated: true, completion: { [weak self] in
```

Comment:
(This line is fine, I'm commenting about an area above)

This spot is where we dismiss the view programmatically (upon tapping Done or the last Next). Users can also swipe to dismiss. The hook for receiving that swipe event is above, where I'm already using it for some logging. Can you add this same call below line 346?

```
    public func presentationControllerWillDismiss(_ presentationController: UIPresentationController) {
        viewModel?.logYearInReviewDidTapDone()
        (self.navigationController as? RootNavigationController)?.turnOffForcePortrait()
    }
```